doc/grammar.txt

```diff
@@ -29,6 +29,7 @@ exprList = expr ^+ comma
 exprColonEqExprList = exprColonEqExpr (comma exprColonEqExpr)* (comma)?
 dotExpr = expr '.' optInd (symbol | '[:' exprList ']')
 explicitGenericInstantiation = '[:' exprList ']' ( '(' exprColonEqExpr ')' )?
+dotLikeExpr = expr DOTLIKEOP optInd symbol
```

Comment:
Unreferenced grammar rule. (My grammar nanny could have picked this up...)

Reply:
but neither are the above 2 rules... I'm just following the rest of the code here

Reply:
in a followup PR, i can remove these 3 rules (pre-existing issue):
```nim
  #| dotExpr = expr '.' optInd (symbol | '[:' exprList ']')
  #| explicitGenericInstantiation = '[:' exprList ']' ( '(' exprColonEqExpr ')' )?
  #| dotLikeExpr = expr DOTLIKEOP optInd symbol
```

Reply:
=> https://github.com/nim-lang/Nim/pull/18746